ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1657,6 +1657,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-confidential-trustedlaunch-f28
```

Comment:
Would you mind to add those tests to 4.15, too ?

Reply:
sure, added.